conf.yaml

```diff
@@ -383,10 +383,14 @@ contents:
                 repo:   elasticsearch
                 path:   buildSrc/src/main/resources/
                 exclude_branches:   [ master, 7.x, 6.1, 6.0, 5.6, 5.5, 5.4, 5.3, 5.2, 5.1, 5.0, 2.4, 2.3, 2.2, 2.1, 2.0, 1.7, 1.6, 1.5, 1.4, 1.3, 0.90 ]
```

Comment:
7.14 needs to be added here

Reply:
Thanks!  Addressed in 275c1f1fd451b5b340777452787c5e1e804fcebb